ui/page/transaction_details_page.go

```diff
@@ -202,7 +202,7 @@ func (pg *TransactionDetailsPage) txnBalanceAndStatus(gtx layout.Context) layout
 							layout.Rigid(func(gtx C) D {
 								return layout.Inset{
 									Right: values.MarginPadding5,
```

Comment:
The right margin should be 4dp as well

<img width="405" alt="Screenshot 2021-07-21 at 14 43 37" src="https://user-images.githubusercontent.com/15804688/126498827-a941c997-2c16-423f-ac18-94c25340f0f8.png">